app/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ApplicationRecord
   has_many :bulk_add_batches
   has_many :import_batches
 
-  serialize :permissions, type: Array
+  serialize :permissions, type: Array, coder: YAML
```

Comment:
Does this functionally change anything? I'm not familiar with coder, I'm basically worried that if we're trying to pass anything but yaml at the moment then this might fail? But this comment may be really stupid because I don't know what I'm talking about!